lab-michael/lib/storage.js

```diff
@@ -0,0 +1,55 @@
+
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+
+const storage = {};
+
+exports.fetchAll = function(recipe) {
+  return fs.readdirProm(`${__dirname}/../data/${recipe}/`)
+  .then(data => {
+    return data.map(str =>
```

Comment:
You're close with this one, but rather than just printing the IDs of the objects, you need to now call fetchItem for each of the IDs that you've parsed. 

Would you do that here in fetchAll and return everything, or do that in your routes?